docs/topics/proc_show-debug-logging-openshift-spring-boot.adoc

```diff
@@ -0,0 +1,91 @@
+
+= Show Debug Logging on OpenShift
+
+Use the the `DEBUG` environment variable in your application pod in OpenShift to enable debug logging.
+
+.Prerequisites
+* An application with debug logging.
+* The `oc` CLI client installed.
+
+
+.Procedure
+
+. Use the `oc` CLI client to log into your OpenShift instance.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc login ...
+----
+
+. Deploy your application to OpenShift.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ mvn clean fabric8:deploy -Popenshift
+----
++
+This command uses the Fabric8 Maven Plugin to launch the link:{link-s2i-process}[S2I process] on OpenShift and to start the pod.
+
+. Find the name of your pod and follow the logs to watch it start.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get pods
+....
+$ oc logs -f pod/POD_NAME
+----
++
+IMPORTANT: After your pod has started, keep watching the logs.
+
+. Test your application.
++
+For example, if you had debug logging in the xref:mission-http-api-spring-boot[{name-mission-http-api} booster] to log the `name` variable in the `/api/greeting` method:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get routes
+...
+$ curl $APPLICATION_ROUTE/api/greeting?name=Sarah
+----
+
+. Return to your pod logs and notice there are no debug logging messages in the logs.
+
+. Set the `DEBUG` environment variable to enable debug logging.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get dc
+...
+$ oc set env dc DC_NAME DEBUG=mybooster
```

Comment:
This is just specific to the Node.js debug logging technique and doesn't really translate to Spring Boot directly.